botorch/models/transforms/outcome.py

```diff
@@ -256,11 +256,11 @@ def untransform_posterior(self, posterior: Posterior) -> Posterior:
                 "Standardize does not yet support output selection for "
                 "untransform_posterior"
             )
-        if not self._m == posterior.event_shape[-1]:
-            raise RuntimeError(
-                "Incompatible output dimensions encountered for transform "
-                f"{self._m} and posterior {posterior.event_shape[-1]}"
-            )
+#         if not self._m == posterior.event_shape[-1]:
+#             raise RuntimeError(
+#                 "Incompatible output dimensions encountered for transform "
+#                 f"{self._m} and posterior {posterior.event_shape[-1]}"
+#             )
```

Comment:
what exactly is going on here? Is this for dealing with the flattened outputs for the HOGP models?

Reply:
Not quite, in the Matheron MTGP implementation, the event shape is `n_train + n_test` (plus batching) which is not the transforms `_m`.